# HG changeset patch
# User Francois Bissey <francois.bissey@canterbury.ac.nz>
# Date 1304306969 -43200
# Node ID 75a0da5d513ca9cf2944c1ec3c5acfdfe0c00bfa
# Parent  018f30f90e4c9d34d6fc81f5e601d47b5dfa5287
trac #10764: fix deprecations warning listed in trac #10493 updated after some were fixed in another ticket.

diff --git a/sage/ext/gen_interpreters.py b/sage/ext/gen_interpreters.py
--- a/sage/ext/gen_interpreters.py
+++ b/sage/ext/gen_interpreters.py
@@ -3042,7 +3042,7 @@
 # Automatically generated by ext/gen_interpreters.py.  Do not edit!
 
 include "../stdsage.pxi"
-from python_object cimport PyObject
+from cpython cimport PyObject
 cdef extern from "Python.h":
     void Py_DECREF(PyObject *o)
     void Py_INCREF(PyObject *o)
@@ -3182,7 +3182,7 @@
         w(je("""
 # Automatically generated by ext/gen_interpreters.py.  Do not edit!
 
-from python_object cimport PyObject
+from cpython cimport PyObject
 
 from sage.ext.fast_callable cimport Wrapper
 {% print s.pxd_header %}
@@ -3343,7 +3343,7 @@
             sage: print rdf_wrapper
             # Automatically generated by ext/gen_interpreters.py.  Do not edit!
             include "../stdsage.pxi"
-            from python_object cimport PyObject
+            from cpython cimport PyObject
             cdef extern from "Python.h":
                 void Py_DECREF(PyObject *o)
                 void Py_INCREF(PyObject *o)
diff --git a/sage/rings/polynomial/polynomial_real_mpfr_dense.pyx b/sage/rings/polynomial/polynomial_real_mpfr_dense.pyx
--- a/sage/rings/polynomial/polynomial_real_mpfr_dense.pyx
+++ b/sage/rings/polynomial/polynomial_real_mpfr_dense.pyx
@@ -4,8 +4,8 @@
 include "../../ext/stdsage.pxi"
 include "../../ext/interrupt.pxi"
 
-from python_int cimport PyInt_AS_LONG
-from python_float cimport PyFloat_AS_DOUBLE
+from cpython cimport PyInt_AS_LONG
+from cpython cimport PyFloat_AS_DOUBLE
 
 from sage.structure.parent cimport Parent
 from polynomial_element cimport Polynomial
